src/scss/components/bars-rulers.scss

```diff
@@ -5,23 +5,25 @@
 @use '../themes/default' as default;
 
 $path2e-bar-color: rgba(51, 51, 51, 1);
+$width-1px: 1px;
```

Comment:
why not give it a descriptive name like `path2e-bar-width`?

Reply:
Honest answer? I was rushing. :(